### ISSUE TITLE
(Feature Request) Is feathers team interested in building a Feathers-Dashboard BaaS?

### ISSUE THREAD

**Issue by jaeko44** [feathersjs/feathers#399]
Hello Feathers Team,

I have been using Feathers recently to power my application, migrating my app from Firebase into Feathers. Feathers is of course a much powerful alternative to fire base, however it's a much more complicated setup. 

Would you guys be interested in launching a new project (with me as a large contributor) to build a Firebase like Dashboard that is deployed with the feathers server on a different port where developers can view their application's database in realtime (and other stats such as connections, etc.), and build the app in a more interactive BaaS style? For example they can use the UI to easily add new datastores (services), and customize their hooks in a more friendly way (drag and drop prebuilt hooks, or add custom ones) etc.

**Comment by kethan** [comment]
Hi I am too interested in creating like that.. I wanted to create like www.cloudboost.io

**Comment by daffl** [comment]
We were bouncing some ideas for something like this around but there aren't any concrete plans yet. I think the best way if someone is interested in this would be to start a project to sketch out some of the ideas of how it might look and work.

We're definitely open for further discussion (hit us up on [Slack](http://slack.feathersjs.com/), too) but I am going to close this issue since it is a very broad topic which might be hard to follow in a single issue.
